README.md

```diff
@@ -20,6 +22,34 @@ Note that `node-gyp` must be installed. See the
 [node-gyp](https://github.com/nodejs/node-gyp) documentation for more
 information.
 
+### npm
+
+It is not recommended to install `hsd` from npm's repositories
+but it is still possible to install using `npm` using a remote
+`git` repository. To install a particular version, a git tag can
+be used. It is recommended to use the latest version.
```

Comment:
nit: latest _tagged_ release.

Assuming we mention that any git ref can be used, we can emphasize to use the latest tagged release.